### DISCUSSION TITLE
A problem occurred evaluating project ':cloud_firestore'.

### DISCUSSION THREAD

**Discussion by KhaledAhmedMohammed** [firebase/flutterfire#9895] [Q&A]
I am trying to use cloud firestore in my app s I added it to pubspec.yaml and ran `flutter pub get` but when I run my app an error happens
 
FAILURE: Build failed with an exception.

* Where:
Build file 'C:\Users\Dell\flutter\.pub-cache\hosted\pub.dartlang.org\cloud_firestore-4.0.4\android\build.gradle' line: ??

* What went wrong:
A problem occurred evaluating project ':cloud_firestore'.
> Could not load compiled classes for script 'C:\Users\Dell\flutter\.pub-cache\hosted\pub.dartlang.org\cloud_firestore-4.0.4\android\user-agent.gradle' from cache.

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 1m 53s
Exception: Gradle task assembleDebug failed with exit code 1

**Comment by TheSilentOwl** [comment]
I am also having the same problem and didn't come to a solution yet.
